src/Symfony/Bundle/FrameworkBundle/Command/SecretsListCommand.php

```diff
@@ -70,6 +74,11 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         }
 
         $secrets = $this->vault->list($reveal);
+
+        if (null !== $name = $input->getArgument('name')) {
+            $secrets = \array_key_exists($name, $secrets) ? [$name => $secrets[$name]] : [];
```

Comment:
perhaps unwanted, but what about leveraging `levenshtein` logic here?

Reply:
Hm. We could maybe do that to implement "did you mean ...?"

But let's do that in a different PR.